app/hooks/useScreenWidth.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
```

Comment:
You might hate my pedantry but might as well stick with kebab case naming convention for file names